mesos-client/src/main/scala/mesosphere/mesos/client/StrictLoggingFlow.scala

```diff
@@ -0,0 +1,10 @@
+package mesosphere.mesos.client
+
+import akka.NotUsed
+import akka.stream.scaladsl.Flow
+import com.typesafe.scalalogging.StrictLogging
+
+// TODO - move somewhere shared
```

Comment:
What does this mean? Why not do it in this diff?

Reply:
I just removed the TODO. I guess we can move it to somewhere shared later but it is fine for now.